CPS-xxxx/README.md

```diff
@@ -0,0 +1,122 @@
+---
+CPS: xxxx
+Title: Register of CBOR Tags for Cardano Data structures
+Status: Open
+Category: Meta
+Authors:
+  - Steven Johnson <steven.johnson@iohk.io>
+Proposed Solutions:
+  - https://github.com/cardano-foundation/CIPs/pull/752
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/751
+Created: 2024-01-24
+License: CC-BY-4.0
```

Comment:
```suggestion
```
Please remove!
adding to the header will break downstream tooling

Reply:
yes @stevenj please do (see https://github.com/cardano-foundation/CIPs/pull/751#discussion_r1483370143)

Reply:
Updated as requested